model/strand.rb

```diff
@@ -43,10 +43,13 @@ def self.lease(id)
 RETURNING lease, exitval IS NOT NULL AS exited
 SQL
     return false unless affected
-    lease = affected.fetch(:lease)
+    lease_time = affected.fetch(:lease)
+    exited = affected.fetch(:exited)
+
+    Clog.emit("obtained lease") { {lease_acquired: {time: lease_time, exited: exited}} }
```

Comment:
Should we add the current label to it?

Reply:
1) Not available in this result set
2) Wouldn't catch multiple transitions done in one lease, note that one lease is taken for several (short) transitions:
https://github.com/ubicloud/ubicloud/blob/f0db6a863de01ec03d621d0ef219eb588a28f6d9/model/strand.rb#L147-L156

Reply:
(But we should have it somewhere, see elsewhere, in the update to this patch)